cypress/integration/brewery/TableParameters-invalid_files.spec.js

```diff
@@ -26,33 +87,47 @@ describe('Table parameters invalid files operations', () => {
     stub.stop();
   });
 
-  it('can import invalid files and display the errors panel', () => {
-    const checkErrorsPanel = () => {
-      const expectedErrors = [
-        { summary: 'Missing fields', loc: 'Line 1' },
-        { summary: 'Incorrect int value', loc: 'Line 2 , Column 1 ("age")' },
-        { summary: 'Incorrect bool value' },
-        { summary: 'Incorrect enum value' },
-        { summary: 'Incorrect date value' },
-        { summary: 'Incorrect number value' },
-        { summary: 'Incorrect int value' },
-        { summary: 'Incorrect bool value' },
-        { summary: 'Incorrect enum value' },
-        { summary: 'Incorrect date value' },
-        { summary: 'Incorrect number value' },
-      ];
-      BreweryParameters.checkCustomersErrorsPanelFromList(expectedErrors);
-    };
+  it('can import invalid files with few or invalid parameters and display the errors panel', () => {
     ScenarioSelector.getScenarioSelectorInput().should('have.value', DEFAULT_SCENARIOS_LIST[0].name);
     ScenarioParameters.expandParametersAccordion();
     BreweryParameters.switchToCustomersTab();
     BreweryParameters.getCustomersImportButton().should('be.visible');
     BreweryParameters.getCustomersExportButton().should('be.visible');
     BreweryParameters.getCustomersErrorsPanel().should('not.exist');
     BreweryParameters.importCustomersTableData(CSV_INVALID_FILE_PATH);
-    checkErrorsPanel();
+    BreweryParameters.checkCustomersErrorsPanelFromList(expectedErrorsFewCSV);
     BreweryParameters.importCustomersTableData(XLSX_INVALID_FILE_PATH);
-    checkErrorsPanel();
+    BreweryParameters.checkCustomersErrorsPanelFromList(expectedErrorsFewXLSX);
+    ScenarioParameters.discard();
+    BreweryParameters.getCustomersErrorsPanel().should('not.exist');
+  });
+
+  it('can import invalid files with few, invalid or much parameters and display the errors panel', () => {
+    ScenarioSelector.getScenarioSelectorInput().should('have.value', DEFAULT_SCENARIOS_LIST[0].name);
+    ScenarioParameters.expandParametersAccordion();
+    BreweryParameters.switchToCustomersTab();
+    BreweryParameters.getCustomersImportButton().should('be.visible');
+    BreweryParameters.getCustomersExportButton().should('be.visible');
+    BreweryParameters.getCustomersErrorsPanel().should('not.exist');
+    BreweryParameters.importCustomersTableData(CSV_TOO_MUCH_PARAMETERS);
+    BreweryParameters.checkCustomersErrorsPanelFromList(expectedErrorsMuchCSV);
+    BreweryParameters.importCustomersTableData(XLSX_TOO_MUCH_PARAMETERS);
+    BreweryParameters.checkCustomersErrorsPanelFromList(expectedErrorsMuchXLSX);
+    ScenarioParameters.discard();
+    BreweryParameters.getCustomersErrorsPanel().should('not.exist');
+  });
+
+  it('can import invalid files with only the name parameter and display the errors panel', () => {
```

Comment:
IMO, we don't need to create two new tests, because:
- there are not many cases to check (too few/too many columns, unexpected empty fields, invalid types) and almost all tests check at least two of them
- the webapp state after importing an invalid file does not prevent to import another file

We can thus write a single `it` block, import all files, check for expected errors after each import, discard changes and check that the error panel has been removed.